Multisig/UI/Transaction/ExecuteTransaction/ReviewExecutionViewController.swift

```diff
@@ -633,9 +633,11 @@ class ReviewExecutionViewController: ContainerViewController, PasscodeProtecting
                 switch result {
                 case .failure(let error):
                     self.submitButton.state = .normal
+                    Tracker.trackEvent(.relayUserFailure)
                     self.didSubmitFailed(error)
 
                 case .success:
+                    Tracker.trackEvent(.relayUserSuccess)
```

Comment:
What is the difference between `.relayUserSuccess` and `.successTxRelay`? Do we need to send both?

Reply:
Apparently `.successTxRelay` is a screen event and `.relayUserSuccess` is a user event. 

Do we need the user event?